### DISCUSSION TITLE
Cannot access to the Traefik Dashboard

### DISCUSSION THREAD

**Discussion by thibaudbrg** [htpcBeginner/docker-traefik#253] [Support]
Hey! I still have a problem which is not the least... I can't access the dashboard...
I configured (by doing a near copy and paste of your docker-compose.yml available on GitHub) traefik2, watchtower, docker-gc and cf-companion. My sensitive data is with Docker Secrets. I configured my domain name with cloudflare, and my CNAMEs are automatically generated by cf-companion.
On my router, ports 80, 443 and 8080 are open and redirects to my Synology IP-Address. I've got a valid Let's Encrypt certificate in the acme.json file. Also thanks to a task, nginx does not uses ports 80 and 443 anymore so there are free to use.
In the logs, no errors, no warnings apppear...

However I can't access the dashboard. With: 
traefikh.$DOMAINENAME0/dashboard/    

Here are some pictures of my configuration if it can help.

PS: Why when my CNAMEs are created automatically, I have a home.$DOMAINENAME0? Where does the "home."
Thank you for your help!

![Screenshot 2022-03-04 094657](https://user-images.githubusercontent.com/73235385/156746629-0d18c343-e4bd-44d5-ab3c-a8d1327430d6.png)
![Screenshot 2022-03-04 095639](https://user-images.githubusercontent.com/73235385/156746672-df946b3f-a4d9-4490-ae3f-387eb73c7283.png)
![Screenshot 2022-03-04 095704](https://user-images.githubusercontent.com/73235385/156746690-4954a859-13c2-4c90-aa9a-f4d6c431299b.png)
![Screenshot 2022-03-04 105335](https://user-images.githubusercontent.com/73235385/156746700-43876660-97e3-4120-839c-10c8a800dc39.png)
![Screenshot 2022-03-04 105658](https://user-images.githubusercontent.com/73235385/156746702-feac5260-1b9f-46e2-a61d-52e49b6bbaa7.png)

**Comment by uasdj25** [comment]
Have you tried accessing the traefik dashboard outside of the URL?  Have you tried localhost:8080?

The home subdomain that you see is a result of the 'TARGET_DOMAIN=' value in the compose file under cf-companion.

If you can access the traefik dashboard locally (as long as the port is published), then try to manually create a CNAME DNS record in cloudflare called 'home'.  That CNAME needs to reference your root domain (A record).  I didn't see it in your list, and that will need to exist I believe in order for your pointer to be associated correctly.  At least that's how mine is set up.  I'd start there and see how it goes.

**Reply by thibaudbrg** [reply]
Thanks for your answer, I finally managed to find the errors. See my corresponding issue.
[#254](https://github.com/htpcBeginner/docker-traefik/issues/254)
